rfc/src/rfcs/0012-list.md

```diff
@@ -66,17 +66,37 @@ Totals:
 
 A "Standard Harness" is a `#[proof]` harness, while a "Contract Harness" is a `#[proof_for_contract]` harness.
 
+#### Output Schema
+The formal output schema is:
+```
+Kani Version: float
```

Comment:
We already print something like:
```
Kani Rust Verifier 0.54.0 ([cargo plugin|standalone]) 
```

Reply:
Yes, that's what I meant, but I should have made it match precisely--will fix.